app/src/main/java/com/hedvig/app/util/extensions/FragmentExtensions.kt

```diff
@@ -2,12 +2,26 @@ package com.hedvig.app.util.extensions
 
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.FragmentTransaction
+import androidx.lifecycle.Lifecycle
 import androidx.lifecycle.lifecycleScope
+import androidx.lifecycle.repeatOnLifecycle
+import kotlinx.coroutines.launch
 
 val Fragment.screenWidth: Int
     get() = requireActivity().screenWidth
 
 val Fragment.viewLifecycleScope
     get() = viewLifecycleOwner.lifecycleScope
 
+fun Fragment.repeatOnLifeCycleLaunch(
```

Comment:
This naming doesn't make it obvious which lifecycle scope we're using here - I immediately got concerned that we were using `lifecycleScope` instead of `viewLifecycleScope` here 😅 Perhaps we could rename it to reflect that we're using `viewLifecycleScope`? 🙂